onnxruntime/__init__.py

```diff
@@ -18,4 +18,4 @@
 from onnxruntime.capi import onnxruntime_validation
 onnxruntime_validation.check_distro_info()
 from onnxruntime.capi.session import InferenceSession
-from onnxruntime.capi._pybind_state import RunOptions, SessionOptions, set_default_logger_severity, get_device, NodeArg, ModelMetadata, GraphOptimizationLevel
```

Comment:
get_device is used to check which compilation settings was used to generate the python package. It might be useful to keep it.

Reply:
looks like this should be called 'get_execution_providers' instead.
@pranavsharma, @Ke zhang may comment.